.buildkite/pipeline.yml

```diff
@@ -3,7 +3,6 @@
 env:
   SETUP_GVM_VERSION: 'v0.5.0'
   DOCKER_COMPOSE_VERSION: '1.25.5'
-  DOCKER_REGISTRY: "docker.elastic.co"
   DOCKER_IMAGE: "${DOCKER_REGISTRY}/observability-ci/temp-fleet-server" # needs to rename after tests
```

Comment:
Should not be declared `DOCKER_REGISTRY` here? 
`DOCKER_IMAGE` variable is expanded correctly with the value of "DOCKER_REGISTRY" ?

I would add back `DOCKER_REGISTRY` here for readability. To see how `DOCKER_IMAGE` would be expanded without checking other files.

Reply:
I don't understand why you are seeing DOCKER_REGISTRY here...
https://github.com/sharbuz/fleet-server/blob/b35a235fadd19ef254a26e7838fda17a743592c6/.buildkite/pipeline.yml#L3-L11
It's already moved to pre-command hook.
And I moved it to pre-command hook because I faced an issue with the undefined value of the DOCKER_REGISTRY when I started to use it in the pre-command hook.
let me check it one more time.

Reply:
Yes, exactly it has been moved to the pre-command hook, and in those changesets it has been removed from the pipeline yaml file.

As DOCKER_IMAGE uses DOCKER_REGISTRY env. var to build the value, I would try (if possible) to set DOCKER_REGISTRY here in the pipeline yaml file. That would help to debug or check DOCKER_IMAGE value.

> And I moved it to pre-command hook because I faced an issue with the undefined value of the DOCKER_REGISTRY when I started to use it in the pre-command hook.

I thought these env. vars defined in the `env` section (pipeline yaml file) should be available in the hooks too. In this case, in the pre-command hook. Those hooks are running in the same agent as the steps/commands, and I would expect to have the same environment. Let's see the results of those tests.